src/System.IO.MemoryMappedFiles/tests/Performance/Perf.MemoryMappedFile.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.IO.MemoryMappedFiles.Tests
+{
+    /// <summary>
+    /// Performance tests for the construction and disposal of MemoryMappedFiles of varying sizes
+    /// </summary>
+    public class Perf_MemoryMappedFile
+    {
+        [Benchmark]
+        [InlineData(10000000)]
+        [InlineData(100000000)]
+        public void ctor(int capacity)
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                MemoryMappedFile f1, f2, f3, f4, f5;
+                using (iteration.StartMeasurement())
+                {
+                    f1 = MemoryMappedFile.CreateNew("newTestMap_1", capacity);
+                    f2 = MemoryMappedFile.CreateNew("newTestMap_2", capacity);
+                    f3 = MemoryMappedFile.CreateNew("newTestMap_3", capacity);
+                    f4 = MemoryMappedFile.CreateNew("newTestMap_4", capacity);
+                    f5 = MemoryMappedFile.CreateNew("newTestMap_5", capacity);
```

Comment:
This is going to fail on Unix.  Names are not supported.  It'd be better to just pass null for the name, unless you actually want to test named maps, in which case that could be a separate, Windows-specific test.